base/test.jl

```diff
@@ -305,6 +305,12 @@ function get_test_result(ex)
             terms[i] = esc(terms[i])
         end
         testret = :(eval_comparison(Expr(:comparison, $(terms...))))
+    elseif isa(ex, Expr) && ex.head == :call && ex.args[1] == :isequal
```

Comment:
`ex.args[1] in (:isequal, :isapprox, :≈)` would also be useful if that comes out looking right